src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,28 @@
+.catalog {
+  display: grid;
+  grid-template-columns: 200px;
+  gap: 48px;
+  justify-content: center;
+
+  @media (min-width: 488px) and (max-width: 768px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media (min-width: 768px) and (max-width: 1024px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 200px);
+  }
```

Comment:
you can pass this magic 200px into card-width variable and then use this variable